theme/ckeditor5-image/imagestyle.css

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+:root {
+	--ck-image-spacing: calc(5 * var(--ck-spacing-small));
+	--ck-image-max-width: 50%;
+}
+
+.ck-editor__editable {
+	& .image-style-side,
+	& .image-style-align-left,
+	& .image-style-align-center,
+	& .image-style-align-right {
+		max-width: var(--ck-image-max-width);
```

Comment:
I'm not sure but for center aligned image`max-width` should be `50%`?

Reply:
Check out the current code in SASS. I think I just copy–pasted it.

Reply:
True, just wondering why centered style of image indicate `max-width: 50%` 🤔